README.md

```diff
@@ -0,0 +1,3 @@
+Explain here what the project is all about. 
+This project should allow for a place where python code can be comoditized. 
+It also allows me to learn fundamental coding principles, and build a portfolio for others to see. 
```

Comment:
This looks like a bulled list to me. Can you add bullets please?

* fundamental coding principles
* build a portfolio

Reply:
I did this.